### ISSUE TITLE
Font size not changing in Separator

### ISSUE THREAD

**Issue by mayorgergich** [Clooos/Bubble-Card#219]
Pretty much the title. Here is my code:
```
  - type: custom:bubble-card
    card_type: separator
    name: Lights
    icon: mdi:lightbulb-group
    styles: |
      ha-card { 
        font-size: 2px;
      }
```

However it doesn't change.

**Comment by Clooos** [comment]
Hi, have you tried with with `font-size: 2px !important` ?

**Comment by mayorgergich** [comment]
Yes I have. Here is the preview with the code as well. I'm also trying to figure out how to change the font size in addition to changing the background color in the first section of the card.
![Uploading Screenshot 2023-12-02 at 12.07.45 PM.png…]()
